stories/header.stories.js

```diff
@@ -51,4 +52,13 @@ storiesOf( 'Global', module )
 				</Header>
 			</WithTestRegistry>
 		);
+	} )
+	.add( 'Plugin Header with Help Menu ', ( registry ) => {
+		return (
+			<WithTestRegistry features={ [ 'helpVisibility', 'storeErrorNotifications' ] } registry={ registry }>
+				<Header>
+					<HelpMenu />
```

Comment:
Can we also add a story that includes additional arbitrary menu items? Maybe we should add a story for the `HelpMenu` itself?

Reply:
I think an extra story with it in the plugin header is fine for now, as that's (mainly) where we use it, and it isn't more complicated than a few stories 🤷🏻‍♂️